build.sbt

```diff
@@ -14,9 +14,8 @@ developers := List(
 
 scalaVersion := "2.12.18"
 
-/**
- * should work with later sbt versions as well (tested at least with 1.4.x)
- */
+/** should work with later sbt versions as well (tested at least with 1.4.x)
+  */
```

Comment:
this should be either 3 lines or 1 line (maybe switch to `//`?), but not 2

Reply:
So there are various different doc types https://scalameta.org/scalafmt/docs/configuration.html#docstringsstyle, can you pick the style you like?